css/main.css

```diff
@@ -325,3 +327,58 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.footer {
+  margin: 70px auto;
+  text-align: center;
+  color: #172b4d;
+}
+
+.form {
+  margin: 30px auto;
+  width: 80%;
+  align-items: center;
+}
+
+.form button {
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  padding: 12px;
+  width: 138px;
+  height: 48px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+.form > div {
+  width: 100%;
+}
+
+.form input,
+.form textarea {
+  padding: 12px;
+  width: 100%;
+  margin: 5px;
+}
```

Comment:
- You should consider the implementation of styling the border of the `input` and `textarea` as per the previous reviewer's suggestion. So please try to fix it. Try to follow the given [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design.

- It is important to follow the project core requirements for professional projects, and if we practice from our learning phase, it will be easier in the future at the industry level. Here is an article on [The importance of understanding software requirements](https://www.freecodecamp.org/news/why-understanding-software-requirements-matter-to-you-as-a-software-engineer/) 

***Your Design:***
![Screenshot from 2022-11-24 01-05-41](https://user-images.githubusercontent.com/60083437/203628002-1e100306-6c0b-4ca7-b999-539457e6d8e6.png)

***Given Figma Design:***
![Screenshot from 2022-11-24 00-58-50](https://user-images.githubusercontent.com/60083437/203627550-9bcf6901-c781-494b-acb7-3a55a9a8fb11.png)